doc/whats_new/v0.21.rst

```diff
@@ -52,6 +52,17 @@ Support for Python 3.4 and below has been officially dropped.
   graph, which would add explicitly zeros on the diagonal even when already
   present. :issue:`12105` by `Tom Dupre la Tour`_.
 
+:mod:`sklearn.neighbors`
+......................
+
+- |Fix| Fixed a bug in :class:`sklearn.neighbors` calling functions
+  :func:`NearestNeighbors.kneighbors`,
+  :func:`NearestNeighbors.radius_neighbors`,
+  :func:`NearestNeighbors.kneighbors_graph`,
+  :func:`NearestNeighbors.radius_neighbors_graph`
+  without first fitting should raise `NotFittedError`.
+  :issue:`12279` by :user:`Krishna Sangeeth <whiletruelearn>`.
```

Comment:
The formatting is still a bit messy. You can copy / paste this, it will do:
 
```
- |Fix| Fixed a bug in :class:`neighbors.NearestNeighbors` calling functions
  :func:`kneighbors <neighbors.NearestNeighbors.kneighbors>`,
  :func:`radius_neighbors <neighbors.NearestNeighbors.radius_neighbors>`,
  :func:`kneighbors_graph <neighbors.NearestNeighbors.kneighbors_graph>`, and
  :func:`radius_neighbors_graph
  <neighbors.NearestNeighbors.radius_neighbors_graph>` without first fitting
  should raise ``NotFittedError``. :issue:`12279` by :user:`Krishna Sangeeth
  <whiletruelearn>`.
```

Reply:
Thanks @NicolasHug . I have updated.